src/CommandingOfficers/DefendPeace/misc/Venge.java

```diff
@@ -88,16 +88,10 @@ public static CommanderInfo getInfo()
   }
 
   @Override
-  public GameEventQueue initTurn(MapMaster map)
+  public GameEventQueue receiveTurnEndEvent(Army army, int turn)
   {
-    GameEventQueue events = super.initTurn(map);
-    return events;
-  }
-  @Override
-  public void endTurn()
-  {
-    super.endTurn();
     aggressors.clear();
+    return null;
```

Comment:
`null` instead of an empty `GameEventQueue`? I guess that saves the overhead of creating an empty one, but then the receiver needs to handle null checks.

Reply:
Though I assume we must already be handling those, or this wouldn't be this way.

Reply:
Yup, and also those changes were on the fog branch.